docs/protocol/src/concepts/validators.md

```diff
@@ -3,44 +3,52 @@
+First, the validator could be jailed and slashed for inactivity.  This can happen in any block, not just at an epoch transition.  Jailed validators are immediately removed from the consensus set. The validator's rates are updated to price in the slashing penalty, and are then held constant. Validators jailed for inactivity are not permanently prohibited from participation in consensus, and their operators can re-activate them by re-uploading the validator definition. Stake cannot be delegated to a slashed validator. Stake already contributed to a slashed validator's delegation pool will enter an unbonding period to hold the validator accountable for any byzantine behavior during the unbonding period. Re-delegations may occur after the validator enters the "Inactive" state.
 
-Second, the validator could be displaced from the validator set by another validator with more stake in its delegation pool. The validator is then in the unbonding state.  It does not participate in consensus, and the stake in its delegation pool does not earn rewards (the validator's rates are held constant).  However, the stake in its delegation pool is still bonded.  Undelegations from an unbonding validator are quarantined with an unbonding period that starts when the undelegation was performed, *not* when the validator began unbonding.  Unbonding validators have three possible state transitions:
+Second, the validator could be tombstoned and slashed for byzantine misbehavior.  This can happen in any block, not just at an epoch transition.  Tombstoned validators are immediately removed from the consensus set. Any pending undelegations from a slashed validator are cancelled: the quarantined output notes are deleted, and the quarantined nullifiers are removed from the nullifier set.  The validator's rates are updated to price in the slashing penalty, and are then held constant. Tombstoned validators are permanently prohibited from participation in consensus (though their operators can create new identity keys, if they'd like to). Stake cannot be delegated to a tombstoned validator. Stake already contributed to a tombstoned validator's delegation pool is not bonded (the validator has already been slashed and tombstoned), so undelegations are effective immediately, with no unbonding period and no quarantine.
+
+Third, the validator could be manually disabled by the operator. The validator is then in the disabled state.  It does not participate in consensus, and the stake in its delegation pool does not earn rewards (the validator's rates are held constant).  The stake in its delegation pool will enter an unbonding period at the time the validator becomes disabled. The only valid state a disabled validator may enter into is "inactive", if the operator re-activates it by updating the validator definition.
```

Comment:
cc @avahowell @plaidfinch currently, disabling a validator triggers an epoch boundary, correct? is that desirable? should we have disabled validators only exit at the next scheduled epoch boundary?

when we do flow encryption, the first block or two after a new epoch won't be able to do trading, while we wait for the dkg to occur. could a validator disable and re-enable to trigger as many epoch boundaries as they want?